clang/test/CodeGenSYCL/intel-sycl-ptr-annotation-builtin.cpp

```diff
@@ -0,0 +1,138 @@
+// RUN: %clang_cc1 -fsycl-is-device -triple spir64-unknown-linux -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
+
+// This test checks that using of __builtin_intel_sycl_ptr_annotation results in correct
+// generation of annotations in LLVM IR.
+#include "Inputs/sycl.hpp"
```

Comment:
Well, that is not exactly what https://github.com/intel/llvm/blob/sycl/sycl/doc/developer/ContributeToDPCPP.md#dpc-clang-fe-tests suggests.
You need to include sycl.hpp like
```
#include "sycl.hpp"
```
and add option to command line to include it as a system header:
```
-internal-isystem %S/Inputs
```